node/operations.js

```diff
@@ -46,8 +46,8 @@ function Operations(opts) {
         out: []
     };
     self.requests = {
-        in: {},
-        out: {}
+        in: Object.create(null),
```

Comment:
However, with the truncated prototype chain, property lookups that miss are slightly faster.

```
kris@krismacuber-9:~ ❯❯ node -e 'var start = Date.now(), eta = start + 10000, count = 0; while (Date.now() < eta) { var o = Object.create(null); for (var index = 0; index < 1e4; index++) o[index]; count += 1e4} console.log(count / (Date.now() - start));'
24977.565061322166
kris@krismacuber-9:~ ❯❯ node -e 'var start = Date.now(), eta = start + 10000, count = 0; while (Date.now() < eta) { var o = Object.create(null); for (var index = 0; index < 1e4; index++) o[index]; count += 1e4} console.log(count / (Date.now() - start));'
24542.003795824592
kris@krismacuber-9:~ ❯❯ node -e 'var start = Date.now(), eta = start + 10000, count = 0; while (Date.now() < eta) { var o = {}; for (var index = 0; index < 1e4; index++) o[index]; count += 1e4} console.log(count / (Date.now() - start));'
21638.54218671992
kris@krismacuber-9:~ ❯❯ node -e 'var start = Date.now(), eta = start + 10000, count = 0; while (Date.now() < eta) { var o = {}; for (var index = 0; index < 1e4; index++) o[index]; count += 1e4} console.log(count / (Date.now() - start));'
20401.317759808328
```